Sources/AutorestSwift/Models/OperationGroup.swift

```diff
@@ -13,6 +13,22 @@ public struct OperationGroup: Codable {
 
     public let operations: [Operation]
 
+    enum CodingKeys: String, CodingKey {
+        case operations, key = "$key"
```

Comment:
```suggestion
        case operations
        case key = "$key"
```

This makes it a little cleaner and clearer.